m2cgen/exporters.py

```diff
@@ -19,92 +24,108 @@ def export_to_java(model, package_name=None, class_name="Model", indent=4):
 
     Returns
     -------
-    code : string
+    code : string or None
     """
     interpreter = interpreters.JavaInterpreter(
         package_name=package_name,
         class_name=class_name,
         indent=indent)
-    return _export(model, interpreter)
+    return _export(model, interpreter, output_file)
 
 
-def export_to_python(model, indent=4):
+def export_to_python(model, output_file=None, indent=4):
     """
     Generates a Python code representation of the given model.
 
     Parameters
     ----------
     model : object
         The model object that should be transpiled into code.
+    output_file : string, optional
+        Path to a file in which the generated code should be written.
     indent : int, optional
         The size of indents in the generated code.
 
     Returns
     -------
-    code : string
+    code : string or None
```

Comment:
Let's document what's being returned when. Here and other places.

Reply:
Sure, done!